evaluation/inference/human_eval/run_human_eval.py

```diff
@@ -0,0 +1,65 @@
+import os
+import torch
+import mii
+import numpy
+import argparse
+from deepspeed.accelerator import get_accelerator
+from transformers import pipeline
+from human_eval.data import write_jsonl, read_problems
+from human_eval.evaluation import evaluate_functional_correctness
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--model", "-m", type=str, help="evaluation model name")
+parser.add_argument("--max-tokens", type=int, default=256, help="max new tokens")
+parser.add_argument("--num-samples-per-task", type=int, default=10, help="number of samples to gen/eval per task")
+parser.add_argument("--local_rank", type=int, default=int(os.getenv("LOCAL_RANK", "0")), help="local rank")
+args = parser.parse_args()
+
+def generate_base_completion(problem_prompt: str) -> str:
+    return base_pipe(problem_prompt, do_sample=True)[0]["generated_text"]
+
+def generate_mii_completion(problem_prompt: str) -> str:
+    return mii_pipe(problem_prompt, max_new_tokens=args.max_tokens)[0].generated_text
+
+def generate_samples(generation_function):
+    samples = [
+        dict(task_id=task_id, completion=generation_function(problems[task_id]["prompt"])) for task_id in problems
+        for _ in range(args.num_samples_per_task)
+    ]
+    return samples
+# TODO (lekurile): Move these functions to utils in DSE repo (TBD)
```

Comment:
@lekurile Is this something you want to do before merging?

Reply:
Removed the TODO, will save this for a future PR. Just want to get the example in for now.